recipes/certbot-apache/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "certbot-apache" %}
+{% set version = "1.15.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/certbot-apache-{{ version }}.tar.gz
+  sha256: a705fa30dcc242914b68d905377f56d0f31a54aac95f3663fbc31b5e467aa9a1
+
+build:
+  number: 0
+  # https://github.com/hercules-team/augeas/issues/612
+  skip: true  # [win]
```

Comment:
These look like they could be noarch python (it’s fine if the windows ci fails with missing dependencies)

Reply:
Done!